cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js

```diff
@@ -0,0 +1,114 @@
+describe('non common configs on location response block and apply undo/redo to changes', () => {
+    beforeEach(() => {
+    cy.createFlow({
+    label: 'Undo/Redo location response configs ',
+    })
+ }) 
+ it('should configure and test the block', () => {
+    const blockConfigs = {
+        label: 'your location',
+            name: {
+                beforeEdit: 'your_location',
+                afterEditPostFix: '_updated',
+            },
+            resources: {
+                ivr: ' 02_flowers_for_albert.mp3',
+                sms: 'Kindly enter your location',
+                ussd: 'Kindly enter your location',
+            },
+        
+        threshold: 
+        {
+           accuracy_threshold:5,
+           accuracy_timeout:120
+         },
+         tags: [
+            {
+               name: 'location_tag',
+               }
+           ]
+        
+    }
+const getStore = () => cy.window().its('store')
+const flowModuleState = () => getStore().its('state.flow')
+const flowsListState = () => flowModuleState().its('flows')
+cy.addBlock(['Content', 'Open Response']).then((LocationBlockUuid) => {
+    const firstBlockState = () => flowsListState().its('[0].blocks[0]')
+    const firstResourcesState = () => flowsListState().its('[0].resources[0]')
+
+// ####### Block label & name
+cy.get('[data-cy="label--editor"]')
+.find('textarea')
+.type(blockConfigs.label)
+firstBlockState().its('label').should('equal', blockConfigs.label)
+firstBlockState().its('name').should('equal', blockConfigs.name.beforeEdit)
+cy.get('[data-cy="name-editor--edit-btn"]').click()
+cy.get('[data-cy="name-editor--input"]')
+.type(blockConfigs.name.afterEditPostFix)
+cy.get('[data-cy="name-editor--save-btn"]').click()
+firstBlockState().its('name')
+.should('equal', `${blockConfigs.name.beforeEdit}${blockConfigs.name.afterEditPostFix}`)
+
+// ####### Resource UIs
+cy.get('[data-cy="audio-library-search--btn"]').click()
+cy.contains('.dropdown-item', blockConfigs.resources.ivr).click()
+cy.get('[data-cy="SMS-resource-variant-text--editor"]')
+.find('textarea')
+.type(blockConfigs.resources.sms)
+cy.get('[data-cy="USSD-resource-variant-text--editor"]')
+.find('textarea')
+.type(blockConfigs.resources.ussd)
+firstResourcesState().its('values').should((values) => {
+expect(values).to.have.length(3)
+
+// the order follows how we set the flow modes from createFlow() command
+expect(values[0].value).to.equal(blockConfigs.resources.sms)
+expect(values[1].value).to.equal(blockConfigs.resources.ussd)
+expect(values[2].value).to.contains('.mp3')
+})
+
+//##### threshold & validating if the values are saved properly#####
+    cy.get('[data-cy="accuracy-threshold"]')
```

Comment:
Hi @safydy-r 
Please i could not hit this selector cy.get('[data-cy="accuracy-threshold"]') in the location script eventhough I added it to the .vue

Reply:
hey Pascal, I will resume this

Reply:
@pascaleli , you need to find the appropriate input or textarea first. See my change here https://github.com/FLOIP/flow-builder/pull/499/files#diff-dd9459d1e5ea3e9abde204db168933e5f8e06f467b7b9cfa4b99948b72ddeae1R29